package.json

```diff
@@ -16,7 +16,8 @@
     "cms:down": "docker compose -f docker-compose.services.yml -f docker-compose.cms.yml down",
     "e2e:up": "docker compose -f docker-compose.services.yml -f docker-compose.e2e.dev.yml up -d",
     "e2e:down": "docker compose -f docker-compose.services.yml -f docker-compose.e2e.dev.yml down",
-    "dev": "node -r ./startup/index.js node_modules/.bin/next dev",
+    "prebuild": "yarn generate",
```

Comment:
ok so I know this `prebuild` script is running when `yarn build` is run because I see it, but I was also looking for documentation about this and found from https://yarnpkg.com/advanced/lifecycle-scripts: 

> ...we intentionally don't support arbitrary pre and post hooks for user-defined scripts (such as prestart). This behavior, inherited from npm, caused scripts to be implicit rather than explicit, obfuscating the execution flow.

this makes me a little nervous we're relying on an unsupported feature to build the app, so I think it might be worth adding a `RUN yarn prebuild` to our Docker file before `yarn build`

Reply:
done! glad you found that, i had no idea